### ISSUE TITLE
Uncaught Error: Invariant Violation: _registerComponent(...): Target container is not a DOM element.

### ISSUE THREAD

**Issue by shaikhussian** [clayallsopp/relay-101#3]
when i run this code i got this error 
Uncaught Error: Invariant Violation: _registerComponent(...): Target container is not a DOM element.
please give solution for it

**Comment by clayallsopp** [comment]
Do you get this issue when cloning the repo?

**Comment by shaikhussian** [comment]
Yes,i cloned your repo and after running the project, I am getting the above mentioned error.when i console.log(mountNode) in index.js i am getting null in console.

**Comment by clayallsopp** [comment]
What's the output of `document.body.childNodes` ? i.e. a screenshot from the debuggger

**Comment by jigarrdesai** [comment]
i got the same issue and bellow is the screen shot of document.body.childNodes

**Comment by jigarrdesai** [comment]
![screen shot 2016-01-05 at 9 11 11 pm](https://cloud.githubusercontent.com/assets/7735437/12121845/9c786696-b3fd-11e5-961a-9d9974764f47.png)

**Comment by clayallsopp** [comment]
That's when you open [http://localhost:8080/webpack-dev-server/](http://localhost:8080/webpack-dev-server/) ? Do you have any errors in your shell? (running `npm start`, right?)

what version of chrome is that? do you have any extensions running that would alter the DOM?

**Comment by jigarrdesai** [comment]
gr8, actually stop-reclame extension alter the DOM. 
Tx...

**Comment by clayallsopp** [comment]
@jigarrdesai great, so it's working for you now?

**Comment by jbw** [comment]
@clayallsopp I get this error still in Chrome (55.0.2883.75) and IE11. The container div is also not present in the dom rendered. Something is removing it?
